### ISSUE TITLE
Qualitative differences when running the examples

### ISSUE THREAD

**Issue by m3at** [wkentaro/chainer-mask-rcnn#33]
Hi, first thanks a ton for that clean implementation of Mask-RCNN, it's very helpful! :slightly_smiling_face: 

When trying to reproduce the results shown in the inference example, I get qualitatively worst results (see bellow).

I'm running this on a 1070Ti on Ubuntu 18.04, with:
```
Platform: Linux-4.15.0-39-generic-x86_64-with-Ubuntu-18.04-bionic
Chainer: 5.0.0
NumPy: 1.15.4
CuPy:
  CuPy Version          : 5.0.0
  CUDA Root             : /usr
  CUDA Build Version    : 9010
  CUDA Driver Version   : 10000
  CUDA Runtime Version  : 9010
  cuDNN Build Version   : 7102
  cuDNN Version         : 7102
  NCCL Build Version    : 2115
iDeep: Not Available

chainercv: 0.11.0
numpy: 1.15.4
fcn: 6.4.8
```
It seems that you developed this on Chainer 4.0, but I could not find in the release notes what change in v5 could make this kind of impact.
Do you have any ideas about what specific change could have affect the results?

Alternatively, some other things could be happening on my side that impact the results, I will investigate as soon as I get the time this week and report it here.

##### Weights from Google Drive
![using_gdrive_weights](https://user-images.githubusercontent.com/3440771/49340658-77c12f80-f686-11e8-9b7f-210b92b4a796.jpg)
##### Weights from Caffe2 (image 1)
![using_caffe_weights_2](https://user-images.githubusercontent.com/3440771/49340657-77289900-f686-11e8-9085-36e36c0eacbf.jpg)
##### Weights from Caffe2 (image 2)
![using_caffe_weights](https://user-images.githubusercontent.com/3440771/49340659-77c12f80-f686-11e8-9411-0c3f4687aa81.jpg)

**Comment by wkentaro** [comment]
Sorry. I have no idea at the moment.

**Comment by wkentaro** [comment]
@m3at Did you find anything that might cause this?

**Comment by wkentaro** [comment]
I've found that the problem is caused by some commit between 4.5.0 to 5.1.0, and the problem is fixed in 5.2.0.
I will filter 5.0.0 and 5.1.0 from requirements.txt in #37

**Comment by m3at** [comment]
> @m3at Did you find anything that might cause this?

Sorry I had no opportunity to dig into it.

Thanks for finding the issue 👍
